tests/Doctrine/Tests/Persistence/Mapping/ClassMetadataFactoryTest.php

```diff
@@ -156,9 +156,12 @@ public function testWillIgnoreCacheEntriesThatAreNotMetadataInstances(): void
 
     private function getArrayCache(): Cache
     {
-        return class_exists(DoctrineProvider::class)
+        $cache = class_exists(DoctrineProvider::class)
             ? DoctrineProvider::wrap(new ArrayAdapter())
             : new ArrayCache();
```

Comment:
```suggestion
            : new Symfony\Component\Cache\DoctrineProvider(new ArrayAdapter());
```
What about using `Symfony\Component\Cache\DoctrineProvider`? So there is no need to ignore the erros?

Reply:
See symfony/symfony#40908

Reply:
to me, if you want to avoid such rule, the proper solution is to bump the min version of doctrine/cache to 1.11+ so that `DoctrineProvider` always exists.

Reply:
@stof That would be a heavy change for a bugfix release. On the 2.2.x branch, this bump has happened already, so we can revert the compat code once this has been merged up.